### DISCUSSION TITLE
Timestamped Comments / Annotations

### DISCUSSION THREAD

**Discussion by WatchAndWarrant** [mediacms-io/mediacms#1145] [Feature Requests]
Hi all! 

For context: 

I'm working on a family-media archive that would use mediaCMS to present hundreds of hours of ripped Hi8, Video8, Super8, and even audio tape of vacations, weddings, funerals, graduations, etc to (mostly elderly) family members. 

As you can imagine, in these long-format recordings (most are 90 minutes in length), it's somewhat difficult to actually locate different events -- while I might be able to tag a certain file with things like "beach", "Mary", or "wedding", it's not exactly useful in actually helping people find the designated content within the given file.

Since mediaCMs uses video.js as the player, has anybody looked at using an approach similar to below?

[contently-videojs-annotation-comments](https://github.com/trilogy-group/contently-videojs-annotation-comments)

In my mind, using a workflow like this to attach timestamped comments to a file, paired with indexable and searchable comments (maybe something like [elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html) running in a different container in the stack, if the already implemented search feature can't be made to work) could be a good solution.

Does anybody else see the value of something like this?

To my eye, it seems like this was actually intended to be a feature that was implemented prior to #802, but was then reverted in #803 because it borked commenting all-together, and then was maybe just dropped?

Similar discussions:
Fuzzy Search Results: #919  
Linking to Specific Parts of a Video: #319 

Anyway, I'm not sure if this feature is being worked on or intended to be implemented again, but it would be really really helpful for this project! I fail to think of another straightforward way to make long-format recordings indexable and searchable, anyway. Maybe the other project above might provide some inspiration on how to get it working again? 

Thanks for the time!

Edit: Looks like this is related to #952

**Comment by WatchAndWarrant** [comment]
@mgogoulos Is there an idea on how to resolve #952 and restore this feature?

**Comment by WatchAndWarrant** [comment]
Resolved in #1220 - thank you devs!

**Reply by mgogoulos** [reply]
there's multiple improvements coming soon (not requiring refresh when selecting a timestamp. Plus chapter functionality. Plus basic video editing)
